spec/components/schemas/Payments/PaymentRequestSource.yaml

```
@@ -29,6 +29,7 @@ discriminator:
     qpay: '#/components/schemas/PaymentRequestQPaySource'
     multibanco: '#/components/schemas/PaymentRequestMultibancoSource'
     dLocal: '#/components/schemas/06_PaymentRequestdLocalSource'
+    sofortv2: '#/components/schemas/PaymentRequestSofortV2Source'
```

Isn't this type actually `sofort` and then routed based gateway routing?
```suggestion
    sofort: '#/components/schemas/PaymentRequestSofortSource'
```
In which case we just use the same source for both? It's backward compatible right? 

(same comment for `PaymentResponseSource.yaml`)

---

I agree, it should be just `sofort`